### PR TITLE
Station selection branch fixes

### PR DESCRIPTION
**Pull request by LeoPhh**
- Removed props from button component
- Moved the div into the dropdown component and removed placeholder attribute
- Changed Dropdown component name to StationDropdown since it's more specific now (has a placeholder called 'Select station')
- Fixed issue with user selecting arrival station that is the same as the departure station
     - Now all stations are visible in both dropdown menus, but an error message is shown when either station is not selected or when both stations are the same

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -26,25 +27,56 @@ const App = () => {
...
+                    label = 'Arrival:'
+                    selectableStations = { stationNames }
+                    id = 'arrival-station-selection'
+                />
             </div>
-            
-            <Button text = 'Select Station' onClick = { onSubmit } classes = 'is-danger' disabled = { !departureStation || !arrivalStation }/>
+
+            {
+                disableSubmit() && getDisabledMessage()
```
Comment (emilies03) [inline src/App.tsx:67]: Do we want to be calling DisableSubmit() twice (line 67 and 74)?
Would this make more sense as a State?

**Review by emilies03 — COMMENTED** [review]
Looks great! Good work :) 

Just worth having a quick think about the disableSubmit logic

File: src/App.tsx (hunk 1 above)
Comment (LeoPhh) [reply src/App.tsx]: Fixed this :) made the `disableSubmit` boolean stateful.

**Review by emilies03 — APPROVED** [review]
Looks great!

**Pull request merged**